index.js

```diff
@@ -191,6 +192,42 @@ demo`;
   info: () => {
     info();
   },
+  config: () => {
+    const args = arg({});
+
+    const action = args._[1];
+    const key = args._[2];
+    const value = args._[3];
+
+    if (!action.length) {
+      return invalid("please provide the action to perform on the config");
```

Comment:
super nit, to be consistent with the rest of the invalid messages in this file
```suggestion
      return invalid("Please provide the action to perform on the config.");
```